package.json

```diff
@@ -106,6 +106,7 @@
     "raven": "2.6.1",
     "raven-js": "git://github.com/vladikoff/raven-js.git#customEndpoint-3.13.0",
     "serve-static": "1.13.1",
+    "sjcl": "git://github.com/vbudhram/sjcl.git#with-hkdf",
```

Comment:
isn't sjcl already part of fxa-js-client? If so can we make it a peerDependency?